runtime/queries/javascript/textobjects.scm

```diff
@@ -0,0 +1,36 @@
+(function_declaration
+  body: (statement_block)) @function.around
+
+(function
+  body: (statement_block)) @function.around
+
+(export_statement
+  declaration: (function_declaration) @function.around) 
```

Comment:
Can we move this down to the other `export_statement`? You could even combine the stanzas:

```scm
(export_statement
  declaration: [
    (function_declaration) @function.around
    (class_declaration) @class.around
  ])
```

It's a little out of place here since it's separating three function-specific queries